js/mercado.js

```diff
@@ -751,8 +753,8 @@ module.exports = class mercado extends Exchange {
         await this.loadMarkets ();
         const market = this.market (symbol);
         const request = {
-            'precision': this.timeframes[timeframe],
-            'coin': market['id'].toLowerCase (),
+            'resolution': this.timeframes[timeframe],
+            'symbol': market['base'] + '-' + market['quote'],
         };
         if (limit !== undefined && since !== undefined) {
```

Comment:
also `to` seems mandatory
![image](https://user-images.githubusercontent.com/43336371/193017011-8db50155-371a-4fb4-9bf2-d6f754163231.png)

Reply:
> @ttodua did you check this comment ⬆️ ?

iirc, the last commit I had pushed was to fix that. retry, but if still issue, please tell me what command you are executing, I reproduced that issue?

Reply:
@ttodua alright I'll check it again, next time leave a comment or resolve the comment pls 🙂

Reply:
ah, right! sry.